Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -228,6 +230,37 @@ void Jit64::lXXx(UGeckoInstruction inst)
     BSWAP(accessSize, Rd);
 }
 
+// For optimization in Jit64::WriteBranchWatch_dcbx.
+#if _WIN32  // 64-bit Windows: RSCRATCH_EXTRA is the same as ABI_PARAM1.
+static_assert(ABI_PARAM1 == RSCRATCH_EXTRA);
+#else  // 64-bit Unix / OS X: ABI_PARAM1 is one of the scratch registers.
+static_assert((BitSet32{ABI_PARAM1} & ABI_ALL_CALLER_SAVED) != BitSet32{});
+#endif
+
+// RSCRATCH2 will hold the amount of faked branch watch hits (loop count minus one)
```

Comment:
Is this a precondition or a postcondition? It reads like it's a postcondition, but as far as I can tell from reading the code it's actually a precondition.

Also, like with WriteBranchWatchDestInRSCRATCH, if the caller would add RSCRATCH2 to the bitset instead of the function doing it internally, I think the code would be clearer.